src/server/lib/middleware/helmet.ts

```diff
@@ -35,6 +36,9 @@ const helmetConfig = {
         gaUID.hash, // google analytics id
         CSP_VALID_URI.GOOGLE_ANALYTICS,
         CSP_VALID_URI.CMP,
+        ...(process.env.NODE_ENV === 'production'
```

Comment:
we generally use `getConfiguration()` and check for the guardian `stage` value and one of `DEV`, `CODE`, `PROD`, in this case we should not apply `UNSAFE_EVAL` in `DEV`

Reply:
sorry I think I'm being a bit thick - do we want to include `UNSAFE_EVAL` if `process.env.NODE_ENV !== 'production' || getConfiguration().stage === 'DEV'

Reply:
ah - I did some experimentation and I've pushed what I think you meant :)